h/logging.h

```diff
@@ -147,6 +183,7 @@ typedef struct LoggingDestination_tag{
   char  *name;
   void  *data;          /* used by destination to hold internal state */
   LogHandler handler;
+  LogHandler2 handler2;
```

Comment:
I'm wondering if adding this in the middle can cause issues. Also, do we have code that creates this structure manually (or accesses the fields) and then passes to the functions in `logging.c` or do we always create it via `logConfigureDestinationX`?

Reply:
I've moved it to the end regardless for safety.